go/test/vschemawrapper/vschema_wrapper.go

```diff
@@ -41,6 +42,7 @@ import (
 var _ plancontext.VSchema = (*VSchemaWrapper)(nil)
 
 type VSchemaWrapper struct {
```

Comment:
While we are here, we should some welcoming comments explaining the why of this code